src/app/markdown/markdown.component.ts

```diff
@@ -23,7 +23,7 @@ import {
 export class MarkdownComponent implements OnChanges, OnInit, AfterViewChecked {
 
   @Input() fieldName: string = 'Description';
-  @Input('renderedText') inpRenderedText: string = '';
+  @Input('renderedText') inpRenderedText: any = '';
```

Comment:
Can we use `SafeHtml` instead of `any`?

Reply:
No, because is can be a string (if there is no need for sanitation anyway) or a SafeValue.

Reply:
Then can you do it, `string | SafeHtml` ?